test/test-classTetrisMoves.js

```diff
@@ -0,0 +1,485 @@
+var sinon = require('sinon');
```

Comment:
this test is too long. Shorten to 100 lines with all equivalence classes

Reply:
what exactly cut?